restli-tools/src/main/java/com/linkedin/restli/tools/symbol/RestLiSymbolTableProvider.java

```diff
@@ -220,6 +220,11 @@ public SymbolTable getSymbolTable(String symbolTableName)
   @Override
   public SymbolTable getRequestSymbolTable(URI requestUri)
   {
+    if (!requestUri.toString().startsWith("d2://"))
```

Comment:
Excellent catch! Can you please change this check to checking for scheme instead of startsWith and use the d2 constant please?

Reply:
Also may want to add a debug log here.